src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -337,5 +337,26 @@ public static void ToString_Compare_Success()
             c.Port = "";
             Assert.Equal("name=value; $Path=path; $Domain=domain; $Port", c.ToString());
         }
+        
+        [Theory]
+        [MemberData(nameof(HashCodesTestData))]
+        public static void HashCodes_ShouldWorkLikeTuple(string name, string value, string path, string domain, int version)
```

Comment:
@ellismg Ok, sure; there are already tests in there [calling GetHashCode already](https://github.com/jamesqo/corefx/blob/003b12b1c016247e10f1d37f994fc96c88a968f3/src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs#L295), so I'm going to go ahead and delete it.